notifiers/providers/sendgrid.py

```diff
@@ -0,0 +1,623 @@
+                         'to be handled.',
+                'additionalProperties': False,
+                'properties': {
+                    'bcc': {
+                        'type': 'object',
+                        'title': 'This allows you to have a blind carbon ' +
+                                 'copy automatically sent to the specified ' +
+                                 'email address for every email that is sent.',
+                        'additionalProperties': False,
+                        'properties': {
+                            'email': {
+                                'format': 'email',
+                                'type': 'string',
+                                'title': 'The email address that you would ' +
+                                         'like to receive the BCC',
+                            },
+                            'enable': {
+                                'type': 'boolean',
+                                'title': 'Indicates if this setting is ' +
+                                         'enabled.'
+                            }
+                        }
+                    },
+                    'bypass_list_management': {
+                        'type': 'object',
+                        'additionalProperties': False,
+                        'title': 'Allows you to bypass all unsubscribe ' +
+                                 'groups and suppressions to ensure that ' +
+                                 'the email is delivered to every single ' +
+                                 'recipient.',
+                        'properties': {
+                            'enable': {
+                                'type': 'boolean',
+                                'title': 'Indicates if this setting is enabled.'
+                            }
+                        }
+                    },
+                    'footer': {
+                        'type': 'object',
+                        'additionalProperties': False,
+                        'title': 'The default footer that you would like ' +
+                                 'included on every email.',
+                        'properties': {
+                            'enable': {
+                                'type': 'boolean',
+                                'title': 'Indicates if this setting is enabled.'
+                            },
+                            'text': {
+                                'type': 'string',
+                                'title': 'The plain text content of your ' +
+                                         'footer.'
+                            },
+                            'html': {
+                                'type': 'string',
+                                'title': 'The HTML content of your footer'
+                            }
+                        }
+                    },
+                    'sandbox_mode': {
```

Comment:
i'd consider just making this value `boolean` and handling the conversion via the `_prepare_data()` method